blade-graphics/src/vulkan/mod.rs

```diff
@@ -295,11 +296,15 @@ pub struct ComputeCommandEncoder<'a> {
     device: &'a Device,
     update_data: &'a mut Vec<u8>,
 }
+//Note: we aren't merging this with `ComputeCommandEncoder`
+// because the destructors are different, and they can't be specialized
+// https://github.com/rust-lang/rust/issues/46893
```

Comment:
Didn't know about this one

Reply:
Quite annoying, honestly! It breaks the API symmetry between standalone types and generic specializations.
I've spent some time experimenting with it and had to throw a lot of stuff out, due to this seemingly minor issue.

Reply:
Very surprising it isn't supported :/